app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Send me reset password instructions" %>
```

Comment:
- [ ] **[OPTIONAL]** When I requested to reset my password, the email link is correctly sent to the email, everything works just fine but the view returns the error in the image below.

  You can fix this by adding `data: {turbo: false}` to this line:
  ```
  <%= f.submit "Send me reset password instructions", data: { turbo: false } %>
  ```
  ![image](https://user-images.githubusercontent.com/81468675/206389034-22034dd2-dab3-4067-a350-dd70a6dd39d5.png)

Reply:
Thanks, @julie-ify. I will correct that.